examples/shortest_paths.txt

```diff
@@ -0,0 +1,9 @@
+['A57', 'A55', 'A30', 'A53', 'B1049', 'A19', 'A11', 'B1041', 'B1042']	A57	B1042	9	226.90000000000003	28.362500000000004
```

Comment:
Is not normal that nodes are rendered as a list (i.e. with the parentheses) and not as comma-separated-values? We should change it.